---
layout: project
title: "Metagenomics"
excerpt: "Analysis tools for large high-throughput metagenomics sequencing projects"
modified: 2013-05-31
rank: 1
---

- J. Paulson, O.C. Stein, H. Corrada Bravo, M. Pop. (2013). "Robust statistical
methods for differential abundance analysis of marker gene microbial
survey data". _Nature Methods_, epub ahead of print. doi:10.1038/
nmeth.2658. [Nature Methods](http://www.nature.com/nmeth/journal/v10/n12/full/nmeth.2658.html)

- J. Wagner, F. Chelaru, J. Kancherla, JN. Paulson, V. Felix, A. Mahurkar, H. Corrada Bravo. (2017).
"Metaviz: interactive statistical and visual analysis of metagenomic data". _Biorvix_ https://doi.org/10.1101/105205

- MS. Kumar, EV. Slud, K. Okrah, SC. Hicks, S. Hannenhalli, H. Corrada Bravo. (2017). "Analysis and correction of compositional bias in sparse sequencing count data". _Biorvix_ https://doi.org/10.1101/142851

- JN. Paulson, H. Talukder, H. Corrada Bravo (2017). "Longitudinal differential abundance analysis of microbial marker-gene surveys using smoothing splines". _Biorvix_ https://doi.org/10.1101/099457

- [`metagenomeSeq` Bioconductor package](http://bioconductor.org/packages/release/bioc/html/metagenomeSeq.html)

- [`metaviz` interactive metagenomic visualization](http://metaviz.org)

- [`metavizr` Bioconductor package](http://bioconductor.org/packages/metavizr)

- [`metagenomeFeatures` Bioconductor package](http://bioconductor.org/packages/release/bioc/html/metagenomeFeatures.html)
