---
layout: project
title: "Second-generation sequencing data quality assessment and analysis"
excerpt: "General methods for  quality
assessment of second-generation sequencing data." 
modified: 2013-05-31
rank: 2.1
---

- Okrah, K. and Corrada Bravo, H. (2015). "Shape analysis of high-throughput transcriptomics experiment data". _Biostatistics_ doi: 10.1093/biostatistics/kxv018. [Biostatistics](http://biostatistics.oxfordjournals.org/content/early/2015/05/10/biostatistics.kxv018.abstract)

- H. Corrada Bravo, R.A. Irizarry. "Model-based quality assessment and
  base-calling for second-generation sequencing data" (Novmeber
  2009). _Biometrics_ Published online before print, Novemeber 13, 2009. doi10.1111/j.1541-0420.2009.01353.x [Biometrics](http://www3.interscience.wiley.com/cgi-bin/fulltext/122684525/HTMLSTART)

- Wu, H., Irizarry, R. A.,  Bravo, H. C.
  (2010). "Intensity normalization improves color calling in SOLiD
  sequencing." _Nature method_, 7(5),
336â€“337. [Nat. Methods](http://www.nature.com/nmeth/journal/v7/n5/full/nmeth0510-336.html)

- Niranjan, T. S., Adamczyk, A., Bravo, H.C, Taub,
      M. A., Wheelan, S. J., Irizarry, R., & Wang,
      T. (2011). "Effective detection of rare variants in pooled DNA
      samples using Cross-pool tailcurve analysis". _Genome
      Biology_, 12(9), R93. [Genome Biology](http://genomebiology.com/content/12/9/R93)

- C. Ye, C. Hsiao, H. Corrada Bravo. BlindCall: ultra-fast base-calling of
second-generation sequencing by blind deconvolution (2014).
Bioinformatics, 30 (9),
1214-9. [Bioinformatics](http://bioinformatics.oxfordjournals.org/content/30/9/1214.full)
